src/ckb-daemon/usb_mac.c

```diff
@@ -838,6 +931,8 @@ int usbmain(){
     if(iterator_hid)
         iterate_devices_hid(0, iterator_hid);
 
+    register_mouse_event_tap(mainloop);
```

Comment:
Actually we don't need to test this on older implementations. We can just wrap it around like this
```c
#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 1012
register_mouse_event_tap(mainloop);
#endif
```
based on [the docs](https://developer.apple.com/library/content/documentation/DeveloperTools/Conceptual/cross_development/Using/using.html) and everything should be fine.

What do you think?